### PR TITLE
fix: format time

### PR DESCRIPTION
**Pull request by hengfeiyang**

### CODE REVIEW

File: src/infra/config.rs [hunk 1]
```diff
@@ -213,7 +213,7 @@ pub struct Limit {
 pub struct Compact {
     #[env_config(name = "ZO_COMPACT_ENABLED", default = true)]
     pub enabled: bool,
-    #[env_config(name = "ZO_COMPACT_INTERVAL", default = 600)] // seconds
+    #[env_config(name = "ZO_COMPACT_INTERVAL", default = 60)] // seconds
```
Comment (gaby) [inline src/infra/config.rs:216]: Why the change? Are there any benefits to having smaller compact interval?

Comment (hengfeiyang) [reply src/infra/config.rs:216]: The compactor also handles data retention, when you delete a stream, earlier you need to wait 10 minutes. if you want to recreate the stream, you can't do anything in 10 minutes. So change it to 1 minutes as default, we can quickly do the job like delete streams.

https://docs.zinc.dev/architecture/#compactor

Comment (gaby) [reply src/infra/config.rs:216]: @hengfeiyang Ohhh! That explains a lot! I ran into that issue while trying to get rid of test fields. Deleted the stream and data was getting rejected.

**Review by prabhatsharma — APPROVED** [review]
